src/docs/product/dashboards/widget-library/index.mdx

```diff
@@ -90,3 +99,25 @@ Update "Columns" to add `links` so you can see seen any external links related t
 
 - Columns: `issue, assignee, events, title, links`
 - Sort by: `Priority`
+
+
+### Release health
+
+<Note>
+
+This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.
+If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+</Note>
+
+To monitor the health of your releases over time, you can modify the "Session Health" widget by updating the "Visualization Display" to "Area Chart":
```

Comment:
Because there are only two actions the user has to take to create this widget, it doesn't make sense to have one of those steps be included in the lead-in sentence.
 
```suggestion
To monitor the health of your releases over time, you can modify the "Session Health" widget as follows:
```